java/cdk-custom-resources/src/test/java/sleeper/cdk/custom/WiremockTestHelper.java

```diff
@@ -43,4 +61,65 @@ public static <B extends software.amazon.awssdk.awscore.client.builder.AwsClient
                         AwsBasicCredentials.create(WIREMOCK_ACCESS_KEY, WIREMOCK_SECRET_KEY)))
                 .build();
     }
+
+    /**
+     * Creates a mocked ECS client.
+     *
+     * @param  runtimeInfo wire mocks runtime info
+     * @return             the ECS client
+     */
+    public static EcsClient wiremockEcsClient(WireMockRuntimeInfo runtimeInfo) {
```

Comment:
It might be worth putting these helpers in a separate file that's specific to ECS, something like WiremockEcsTestHelper.

It looks like the original file here is no longer needed as it duplicates WiremockAwsV2ClientHelper.